### DISCUSSION TITLE
Is `mill-bloop` still needed?

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/bloop#1427] [Q&A] [answered]
I see as of 1.3.0 I see that it was deprecated in favor of the [Mill Bloop contrib](http://www.lihaoyi.com/mill/page/contrib-modules.html#bloop).  I was in the process of updating the 2.12.8 -> 2.12.12 (trying to get some things updated) and one of the things I hit on is that the `mill-scalalib`, which is a dependency of `mill-bloop` is no longer published for 2.12. This matters because of the reliance on Ammonite. If we bump to 2.12.12 it will attempt to download the 2.12.12 Ammonite, which doesn't exist for the current old version of `mill-scalalib`. So in order to update the `mill-bloop` module it either needs to:

- Update to 2.13 instead and not cross to 2.12
- Just remove it since it doesn't seem to be used (at least by Metals or Mill folks it seems)
- Just stay on 2.12.8?

Any ideas about what would be best here?

**Comment by tgodzik** [comment, marked as answer]
I think it should be removed right now. Especially that it's available by default in Mill and it might cause additional confusion if it stays in Bloop

**Reply by ckipp01** [reply]
Ok cool. I'll go ahead and continue to try and finish the update to 2.12.12 and then just remove this mill plugin.
